lldb/source/Plugins/Process/elf-core/ProcessElfCore.cpp

```diff
@@ -1034,7 +1041,8 @@ UUID ProcessElfCore::FindBuidIdInCoreMemory(lldb::addr_t address) {
     std::vector<uint8_t> note_bytes;
     note_bytes.resize(program_header.p_memsz);
 
-    byte_read = ReadMemory(program_header.p_vaddr, note_bytes.data(),
+    const lldb::addr_t program_header_vaddr = program_header.p_vaddr + address;
+    byte_read = ReadMemory(program_header_vaddr, note_bytes.data(),
```

Comment:
This fix is being tracked by https://github.com/llvm/llvm-project/pull/117028. Remove and make this dependent on that PR